source/appModules/soffice.py

```diff
@@ -101,6 +107,47 @@ def _getFormatFieldAndOffsets(self,offset,formatConfig,calculateOffsets=True):
 		if backgroundColor:
 			formatField['background-color']=colors.RGB.fromString(backgroundColor)
 
+		if offset == 0:
+			# Only include the list item prefix on the first line of the paragraph.
+			numbering = formatField.get("Numbering")
+			if numbering:
+				formatField["line-prefix"] = numbering.get("NumberingPrefix") or numbering.get("BulletChar")
+
+		return formatField
+
+	def _getFormatFieldAndOffsets(  # noqa: C901
+			self,
+			offset: int,
+			formatConfig: Optional[Dict],
+			calculateOffsets: bool = True
+	) -> Tuple[textInfos.FormatField, Tuple[int, int]]:

```

Comment:
We are using Python 3.11, where collections in standard library are generic, so this can be written as:
```suggestion
	) -> tuple[textInfos.FormatField, tuple[int, int]]:
```

Reply:
Thanks, done in the new version.